docs/guides/vm-architectures/README.md

```diff
@@ -0,0 +1,385 @@
+
+**Key characteristics:** The core principle of this architecture is to run two or more independent, self-contained VictoriaMetrics clusters (from the section about Single AZ) in separate failure domains, such as different Availability Zones or geographic regions. A global, stateless layer is responsible for routing write and read traffic to these clusters. Each participating AZ must be provisioned to handle the entire workload if another AZ fails. 
+
+There are no differences in the VictoriaMetrics clusters' realisation regarding the multi-AZ approach. It can be Active-Active or Active-Passive \- the schema will be the same.  
+
+To ensure reliability, vmagent implements the bulkhead pattern: each destination URL configured via `--remoteWrite.url` is assigned a dedicated data queue and an isolated pool of workers. This isolates the data streams, ensuring that if one storage destination becomes slow or unavailable, it does not impact data delivery to the others.
+
+**Pros:**
+
+* **Disaster Recovery:** The system can survive a complete failure of one cluster's location (AZ or region).  
+* **Isolation:** Incidents, maintenance, or configuration errors in one cluster do not affect the others.
+
+**Cons:**
+
+* **Increased Cost:** You are paying more for the infrastructure (compute, storage, and network). The capacity of vmstorage in each AZ is underutilized, since every AZ must be ready to absorb the full traffic load in case of failure. Overhead is \~100% with 2 AZs (50% utilization). For other components, it is possible to use [VPA](https://kubernetes.io/docs/concepts/workloads/autoscaling) or [HPA](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale). 
+
+**Schema:**
+
+![Multi-AZ Architecture](multi-az-architecture.webp)
+
+### Unavailability Scenarios
+
+**Blast radius:** Availability zone
+
+* **Primary region failure (Active-Passive):** switchover in minutes; stale reads until DNS/load balancer/BGP reroute.
+
+* **Single AZ/cluster failure (Active-Active):** seamless reroute; read results may temporarily differ between clusters if cross-AZ replication lags.
+
+* **Cross-region link failure:**
+
+  * Writes: buffered by vmagent.  
+  * Reads: may return stale data until the link is restored.
+
+## The Hyperscale (Cell-based)
+
+**For whom:** For systems that require extra reliability and scalability across multiple regions and zones.
+
+**Key characteristics:** This architecture is built on two main ideas \- cells and the separation of routing and storage paths
+
+First, we have logical groups of Availability Zones (AZs). Think of these as our data pods. Inside these groups, we deploy our basic clusters. The data within these groups can be either fully replicated (an identical copy in each AZ) or sharded (each AZ holds a distributed portion of the data. For example, in a sharded setup with a replicate data in 3 across 4 cells, each cell will store approximately 75% of all metrics.). 
```

Comment:
please, simplify this paragraph. Full sentences in braces are too much :)

Reply:
solved